packages/wrangler/src/dev/local.tsx

```diff
@@ -89,7 +89,8 @@ export async function localPropsToConfigBundle(
 		inspectorPort: props.runtimeInspectorPort,
 		bindings: props.bindings,
 		workerDefinitions: props.workerDefinitions,
-		assetPaths: props.assetPaths,
+		// TODO @Carmen can we rename here?
```

Comment:
What is the answer here?

Reply:
the answer was yes, and `assetsPath` was renamed to `legacyAssetsPath`, but thank you for spotting the todo comment. I cleaned that up